api/middleware/auth.js

```diff
@@ -13,11 +13,13 @@ module.exports.loggedIn = (req, res, next) => {
   }
 };
 
+module.exports.loggedIn = loggedIn;
```

Comment:
how do you feel about this general pattern: if we assign an exported function to a variable name in a file, then we assign all exported functions to variables (in that file) and then the very last line is the collection of all exports:  `module.exports = { loggedIn, can }`

Reply:
I am on board with that.